spec/support/check.rb

```diff
@@ -8,23 +8,23 @@ def self.included(by)
 
     by.let(:shell) { FakeShell.new }
     by.let(:context) { Hashie::Mash.new }
-    by.subject { described_class.new(shell: shell, context: context) }
+    by.let(:check) { described_class.new(shell: shell, context: context) }
```

Comment:
A named `subject` means we can reset the subject to something else in specs, eg. the following for testing the `#fix` method

```ruby
subject(:fix) { check.fix }
```